src/__tests__/multiple-outputs.test.ts

```
@@ -0,0 +1,395 @@
+    it('should support console + file + network logging pattern', () => {
+      const fileLogs: string[] = [];
+      const networkLogs: Array<{ level: string; message: string; timestamp: string }> = [];
+
+      const fileHandler: LogOutputHandler = (level, message) => {
+        fileLogs.push(`[${level.toUpperCase()}] ${message}`);
+      };
+
+      const networkHandler: LogOutputHandler = (level, message) => {
+        networkLogs.push({
+          level,
+          message: message.replace(/\x1b\[[0-9;]*m/g, ''), // Strip ANSI colors
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)